rpcs3/main.cpp

```diff
@@ -18,6 +19,40 @@ namespace logs
 	void set_init();
 }
 
+static semaphore<> s_init{0};
+static semaphore<> s_qt_init{0};
+static semaphore<> s_qt_mutex{};
+
+[[noreturn]] extern void report_fatal_error(const std::string& text)
+{
+	s_qt_mutex.wait();
+
+	if (!s_qt_init.try_wait())
+	{
+		s_init.wait();
+		static int argc = 1;
+		static char arg1[] = {"ERROR"};
+		static char* argv[] = {arg1};
+		static QApplication app0{argc, argv};
+	}
+
+	QMessageBox msg;
+	msg.setWindowTitle("RPCS3: Fatal Error");
```

Comment:
maybe add tr() here
and down at setText(tr(...))

same could be done in experimental build warning and SSSE3 warning.